libfreerdp/crypto/tls.c

```diff
@@ -1198,9 +1198,10 @@ static BOOL is_accepted_fingerprint(CryptoCert cert, const char* CertificateAcce
 		char* cur = strtok_s(copy, ",", &context);
 		while (cur)
 		{
+			char* subcontext = NULL;
```

Comment:
Same

Reply:
no, the context must be reset for each new `cur`